README.md

```diff
@@ -63,3 +65,33 @@ to document the current shortcomings of this tool.
 
 * Since `garble` forces `-trimpath`, plugins built with `-garble` must be loaded
   from Go programs built with `-trimpath` too.
+
+### Runtime API
+
+The tool adds additional functions to the runtime that can optionally be used to
+hide information during execution. The funcions added are:
+
+```go
+// HidePanics suppresses printing fatal panic messages when hide
+// is true. This behavior can be changed at any time by calling
+// HidePanics again. All other behaviors of panics remains the
+// same.
+func HidePanics(hide bool)
```

Comment:
I wonder if we should make this `hidePanics` too, to clarify that it can't be called direclty like other exported runtime APIs.